packages/react/src/componentsV2/payStream/paystreamPricing/index.tsx

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+const PaystreamPricing = () => <div />;
```

Comment:
I guess this is a temp component and you are going to add code to it later?

Reply:
yes